### ISSUE TITLE
Update notification can show short what's new

### ISSUE THREAD

**Issue by fildunsky** [esteemapp/esteem-surfer#104]
I've just updated 1Password and it was showing nice notification window explaining what's new there before update. More user-friendly so people can decide to update or not if there is something interesting in the update.

Not sure if it's needed though. How do you think?

**Comment by feruzm** [comment]
yes, initial design was to have changelog popup to inform users about changes but we changed that to open changelog post, I still think there is a merit to have those type of popup screens, I think we can make popup bigger and get portion of content from post and display only `body` of the post as changelog, what do you think @talhasch ?

**Comment by fildunsky** [comment]
Yes, maybe we could fetch only small part of that post (bottom one starting from # What's New...) and excluding the post footer.

**Comment by talhasch** [comment]
Electron auto updater allows to read change log from github. We can fetch new release post address and put a "what's new" link linked to it on popup like below

![image](https://user-images.githubusercontent.com/1177676/42320375-d114a6c4-805d-11e8-8e48-d22992f0898d.png)
